app/utils/createPage.js

```diff
@@ -33,7 +33,7 @@ const createPage = ({
   if (loading && !fetched) {
     content = <Loader {...props} />;
   } else if (error) {
-    content = <ErrorComponent error={error} {...props} />;
```

Comment:
Pass the error here. The error is not being used in the default ErrorLayout, but we can pass custom ErrorComponent also.

Reply:
ok, like Error Component can be a higher-order function which uses error in some way and returns ErrorLayout Component depending on somethings.